### ISSUE TITLE
Mirrored Filament Blocks

### ISSUE THREAD

**Issue by SamueldaCostaAraujoNunes** [moggieuk/Happy-Hare#548]
I noticed that, like me, many people use their ERCFs backwards, with the front pointing towards the back of the printer. This allows them to, for example, place filaments on top of the printer. The bad thing is that there is no possibility today of inverting the doors. So that, when you see the front of the printer, the sequence of extruders would be from left to right. Is this feasible?

**Comment by moggieuk** [comment]
I'd certainly entertain a PR for this but it would be very low priority for me to work on... just too much testing.

It's probably not a complex change.  Any reference to `self.gate_selected` and `self.tool_selected` would need to be changed to call function that applied the inversion. But there would be a lot of places to test.

**Comment by SamueldaCostaAraujoNunes** [comment]
Yes, I would probably make a parameter of a macro, inverted gates. And when I was going to get the position of the tool or the gate I would do, NUMBER OF GATES - CURRENT GATE

I'm going to fork the project and test it here

**Comment by SamueldaCostaAraujoNunes** [comment]
I found an existing solution that worked very well:
https://github.com/moggieuk/Happy-Hare/wiki/Tool-and-Gate-Maps#:~:text=MMU_REMAP_TTG%20MAP%3D8%2C7%2C6%2C5%2C4%2C3%2C2%2C1%2C0
